python/tests/parsers/test_main.py

```diff
@@ -146,25 +149,38 @@ def test_send_data_to_db_savesData13fInDb(self):
         """Tests if send_data_to_db saves the Data13f model in the database"""
         company = Company(cik_no="009039443", company_name="Apple Industries", filing_count=0)
         edgar_filing = EdgarFiling(accession_no="78945835", cik_no="009039443",
-                                     filing_date=datetime.fromisoformat("2000-06-11"))
+                                   filing_date=datetime.fromisoformat("2000-06-11"))
         data_13f_table = [Data13f(equity_holdings_id="67896567",
-                                    accession_no='78945835',
-                                    cik_no='009039443',
-                                    name_of_issuer='Agilent Technologies',
-                                    title_of_class='COM',
-                                    cusip='00846U101',
-                                    value='22967078',
-                                    ssh_prnamt='180644',
-                                    ssh_prnamt_type='None',
-                                    put_call='None',
-                                    investment_discretion='SOLE',
-                                    other_manager='None',
-                                    voting_authority_sole='22967078',
-                                    voting_authority_shared='0',
-                                    voting_authority_none='0'
-                                    )
-                            ]
+                                  accession_no='78945835',
+                                  cik_no='009039443',
+                                  name_of_issuer='Agilent Technologies',
+                                  title_of_class='COM',
+                                  cusip='00846U101',
+                                  value='22967078',
+                                  ssh_prnamt='180644',
+                                  ssh_prnamt_type='None',
+                                  put_call='None',
+                                  investment_discretion='SOLE',
+                                  other_manager='None',
+                                  voting_authority_sole='22967078',
+                                  voting_authority_shared='0',
+                                  voting_authority_none='0'
+                                  )
+                          ]
 
         send_data_to_db(company, edgar_filing, data_13f_table)
 
         assert Data13f.query.filter_by(cik_no='009039443').first() == data_13f_table[0]
+
+
+def test_change_sys_excepthook():
+    change_sys_excepthook()
+    assert sys.excepthook is my_handler
```

Comment:
can you reset the handler after testing it? See https://docs.python.org/3/library/sys.html#sys.__excepthook__ for help

Reply:
Reset it in the next line.